services/121-service/src/financial-service-providers/dto/update-financial-service-provider.dto.ts

```diff
@@ -1,16 +1,18 @@
 import { ApiProperty } from '@nestjs/swagger';
 import { IsNotEmpty, IsOptional, IsString } from 'class-validator';
+import { LocalizedString } from 'src/shared/enum/language.enums';
+import { QuestionOption } from 'src/shared/enum/question.enums';
 import { ExportType } from '../../metrics/dto/export-details.dto';
 import { ProgramPhase } from '../../shared/enum/program-phase.enum';
 
 export class UpdateFspAttributeDto {
   @ApiProperty({ example: { en: 'attribute label' } })
   @IsOptional()
-  public label: JSON;
+  public label: LocalizedString;
```

Comment:
Should this also be LocalizedString | null because of the is IsOptional? Or is LocalizedString already nullable?

Same question for the other `@IsOptional()` dto attributes?

Reply:
Excellent question. Indeed they need to be addressed too. Will look into this.